openedx/core/djangoapps/public_api/courses/tests/test_course_import_export.py

```diff
@@ -0,0 +1,416 @@
+"""
+Unit tests for course import and export
+"""
+import copy
+import json
+import logging
+import os
+import shutil
+import tarfile
+import tempfile
+from path import path
+from uuid import uuid4
+
+from django.test.client import Client
+from django.test.utils import override_settings
+from django.conf import settings
+from django.core.urlresolvers import reverse
+
+from xmodule.modulestore.tests.factories import ItemFactory, CourseFactory
+
+from .utils import CourseTestCase
+from student import auth
+from student.roles import CourseInstructorRole, CourseStaffRole
+
+TEST_DATA_CONTENTSTORE = copy.deepcopy(settings.CONTENTSTORE)
+TEST_DATA_CONTENTSTORE['DOC_STORE_CONFIG']['db'] = 'test_xcontent_{}'.format(
+    uuid4().hex
+)
+
+log = logging.getLogger(__name__)
+
+
+def course_url(handler, course_key, kwargs=None):
```

Comment:
Don't you want this to be `**kwargs`? It's the difference between calling
```python
course_url(my_handler, my_course_key, kwargs={"foo": "bar", "num": 123})
```
and calling
```python
course_url, my_handler, my_course_key, foo=bar, num=123)
```